dvc/command/diff.py

```diff
@@ -121,7 +121,7 @@ def run(self):
 def add_parser(subparsers, parent_parser):
     DIFF_DESCRIPTION = (
         "Show changes between commits in the DVC repository,"
-        " or between a commit and the workspace."
+        " or between a commit and the workspace. "
         "Prints the list of paths added, modified, or deleted."
```

Comment:
```
Show added, modified, or deleted data
between commits in the DVC repository,
or between a commit and the workspace
```
to match https://dvc.org/doc/command-reference/diff

I'll edit this in a regular updates PR...

Reply:
(Including zsh autocomplete script.)

Reply:
See #3601